CMakeLists.txt

```diff
@@ -87,9 +87,10 @@ option(WITH_STBTRUETYPEFONT "Build StbTrueTypeFont plugin" OFF)
 option(WITH_STBVORBISAUDIOIMPORTER "Build StbVorbisAudioImporter plugin" OFF)
 option(WITH_STLIMPORTER "Build StlImporter plugin" OFF)
 option(WITH_TINYGLTFIMPORTER "Build TinyGltfImporter plugin" OFF)
+option(WITH_WEBPIMPORTER "Build WebPImporter plugin" OFF)
 
 include(CMakeDependentOption)
-option(BUILD_TESTS "Build unit tests" OFF)
+option(BUILD_TESTS "Build unit tests" ON)
```

Comment:
There still seems to be a few changes to the main CMakeLists.txt here. Adding `WITH_WEBPIMPORTER` is great, but the others should be changed back 👍

Reply:
But now I see I get an error for trying to change CMakeLists.txt (together with other config files) because I don't have "write access".